authui/src/inline-preview.ts

```diff
@@ -89,6 +91,11 @@ export class InlinePreviewController extends Controller {
       outlet?.setValue(value);
     }
 
+    const currentTheme = el.getAttribute("class");
+    if (currentTheme !== customisationMessage.theme) {
+      el.setAttribute("class", customisationMessage.theme);
+    }
```

Comment:
Here you will also remove other classes. Please just add/more remove the class we are targeting.